inc/classes/subscriber/third-party/plugins/class-ngg-subscriber.php

```diff
@@ -39,6 +39,10 @@ public static function get_subscribed_events() {
 	 * @return bool
 	 */
 	public function deactivate_resource_manager() {
+		if( is_admin() ) {
+			return true;
+		}
+		
 		return false;
```

Comment:
```suggestion
		return is_admin();
```

This block of code can be simplified to the suggestion above. `is_admin()` already returns a boolean value, so we can use that directly for the return, and turn 4 lines of code into one 🎉